app/representers/space_representer.rb

```diff
@@ -13,9 +13,12 @@ class SpaceRepresenter < Roar::Decorator
   property :is_private
   property :user_id
 
+  # TODO(matthew): These aren't being used atm. Should we remove them?
```

Comment:
If you make sure they aren't actually being used, I feel like they are for some reason.  Can discuss.